docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.3'
+
+services:
+
+  barc-worker:
+    build: .
+    links:
+      - "redis"
+    environment:
+      - REDIS_HOST=redis
+
+  barc-ws:
+    build: "./webapp/"
+    volumes:
+      - "./webapp/bin/www:/src/bin/www"
+    ports:
+      - "11500:443"
```

Comment:
Where is this internal port number coming from? I don't see any change to the default used in this PR.

Reply:
@wobbals I used port 11500 only for development usage. 
This is an absolutely random port number which shouldn't conflict with other ports on developer's development machine.

Added instructions for developers:

* Open `http://localhost:11500/barc` in your browser to check working
* Use `http://localhost:11500/barc/job` to put your jobs

Reply:
Docker links guest port 11500 (developer's machine) with host port 443 (inside Docker container)

Reply:
Ah, my mistake. I was reading `CONTAINER:HOST` rather than `HOST:CONTAINER` for the port declaration syntax.